src/components/pets/PetFormLayout.js

```diff
@@ -6,7 +6,7 @@ export default styled.form`
   grid-template-rows: auto;
   align-items: center;
   color: var(--secondary);
-  margin: 4px 20px 20px 20px;
+  margin: 4px 12px 12px 12px;
```

Comment:
Same here, there should be a shorthand for that?

    margin: X is expanded to X X X X
    margin: X Y is expanded to X Y X Y (X on top and bottom, Y on sides)
    margin: X Y Z is expanded to X Y Z Y (X on top, Y on sides, Z on bottom)
    margin: X Y Z T defines all four margins explicitly.